adapters/handlers/graphql/schema.go

```diff
@@ -102,6 +104,9 @@ func buildGraphqlSchema(dbSchema *schema.Schema, logger logrus.FieldLogger,
 	func() {
 		defer func() {
 			if r := recover(); r != nil {
+				if entsentry.Enabled() {
+					sentry.CurrentHub().Recover(r)
+				}
```

Comment:
just a suggestion, bc right now in order to add sentry we have to always import 2 packages, maybe it would be better to encapsulate this in `Recover` method in `entsentry` package and then just add:

```
ensentry.Recover(r)
```

Reply:
Great idea!